### PR TITLE
Add TLS functionality based on Prometheus Web module. Switch to logger (as prometheus/common/log is deprecated)

### PR DESCRIPTION
**Pull request by lucian-vanghele**
@Lusitaniae please review my PR and approve it if you agree with the changes.
Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,21 @@ Q. There seem to be missing metrics! I can only see apache_up and apache_cpuload
 
 A. Make sure that you add `?auto` at the end of the scrape_uri.
 
+
+
+## TLS and basic authentication
+
+Graphite Exporter supports TLS and basic authentication. This enables better
```
Comment (Lusitaniae) [inline README.md:163]: Apache Exporter

**Comment by Lusitaniae** [comment]
You may need to update module list, seems to be the reason the build is failing

```
apache_exporter.go:24:2: missing go.sum entry for module providing package github.com/go-kit/kit/log (imported by github.com/Lusitaniae/apache_exporter); to add:
	go get github.com/Lusitaniae/apache_exporter
/go/pkg/mod/github.com/prometheus/common@v0.18.0/promlog/log.go:24:2: missing go.sum entry for module providing package github.com/go-kit/kit/log/level (imported by github.com/prometheus/common/promlog); to add:
	go get github.com/prometheus/common/promlog@v0.18.0
apache_exporter.go:25:2: no required module provides package github.com/go-kit/log/level; to add it:
	go get github.com/go-kit/log/level
/go/pkg/mod/github.com/prometheus/common@v0.18.0/promlog/log.go:25:2: missing go.sum entry for module providing package github.com/pkg/errors (imported by github.com/prometheus/common/promlog); to add:
	go get github.com/prometheus/common/promlog@v0.18.0
apache_exporter.go:31:2: no required module provides package github.com/prometheus/exporter-toolkit/web; to add it:
	go get github.com/prometheus/exporter-toolkit/web
```

**Comment by lucian-vanghele** [comment]
@Lusitaniae - I fixed all the issues, now build was ok!

File: README.md (hunk 1 above)
Comment (Lusitaniae) [reply README.md:163]: this small thing please

Comment (lucian-vanghele) [reply README.md]: oops...

**Comment by dswarbrick** [comment]
I look forward to this being merged and a new release published. The prometheus-apache-exporter package in Debian, which I co-maintain, is currently FTBFS since it depends on the deprecated prometheus/common logrus wrapper.

**Comment by Lusitaniae** [comment]
Thank you @lucian-vanghele 

Just cut new release https://github.com/Lusitaniae/apache_exporter/releases/tag/v0.11.0

**Comment by dswarbrick** [comment]
Thank you both, new Debian package on its way.

One tiny nit: there is one remaining import of `github.com/go-kit/kit/log` in apache_exporter.go which could be replaced with `github.com/go-kit/log`. This is not a big deal though, and does not justify a new tag.

**Comment by Lusitaniae** [comment]
@dswarbrick thank you, I made a new issue

**Pull request merged**